ui/models.py

```diff
@@ -120,14 +120,13 @@ def for_owner(cls, owner):
         return cls.objects.filter(owner=owner)
 
 
-class Video(models.Model):
+class Video(TimestampedModel):
```

Comment:
The default ordering for `Video` is `-created_at`, which is probably still fine, but maybe check with @pdpinch to see if he'd prefer `updated_at` instead.